taxi/models.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=240, unique=True)
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=240, unique=True)
+    country = models.CharField(max_length=240)
+
+    def __str__(self):
+        return self.name
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=240)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name="Manufacturer"
+    )
+    drivers = models.ForeignKey(
```

Comment:
`drivers` should have a many-to-many relationship, change the field to encounter that.